docs/framework/data/adonet/connecting-to-a-data-source.md

```diff
@@ -1,7 +1,7 @@
 ---
 description: "Learn more about: Connecting to a Data Source in ADO.NET"
 title: "Connecting to a Data Source"
-deescription: Learn about Connection objects, used to connect to data sources in ADO.NET. The Connection object you choose depends on the type of data source.
+description: Learn about Connection objects, used to connect to data sources in ADO.NET. The Connection object you choose depends on the type of data source.
```

Comment:
Getting a warning since there is already a key, but not sure which one to keep

Reply:
Thanks. The longer description on line 4 should be kept.

Reply:
OK, deleted the first one